src/index.js

```diff
@@ -184,16 +196,9 @@ export default function wire(contextNamespace, mapToProps={}, mapModelToProps=no
 							rejected[prop] = err;
 							let newState = { rejected };
 							if (CACHE[key]) newState[prop] = CACHE[key];
-							return newState;
-						}).then( newState => {
-							if (this.tracking[prop]!==id) return;
-							delete this.tracking[prop];
-
-							// remove the pending key for this prop if necessary
-							let pending = this.state.pending;
-							if (pending && pending[prop]) newState.pending = removeKeyFromObject(prop, pending);
-							this.setState(newState);
-						});
+							after(newState);
+							return Promise.reject(err);
```

Comment:
Could make this simpler by rejecting an arbitrary promise instead of the chain itself:

```js
.catch( err => {
  // [as before]
  // anonymous rejection just to log to the console:
  Promise.reject(err);
  return newState;
}).then( newState => {
  // [as before]
});
```